po/ru.po

```diff
@@ -186,374 +195,710 @@ msgstr "помечены как устаревшие: "
 msgid "orphans: "
 msgstr "сироты: "
 
-#: src/download.rs:206
+#: src/download.rs:204
 msgid "could not get current directory"
 msgstr "не удалось получить текущий каталог"
 
-#: src/download.rs:210
+#: src/download.rs:208
 msgid "can not get repo packages: asp is not installed"
 msgstr "не удалось получить пакеты из репозитория: не установлен asp"
 
-#: src/download.rs:231
+#: src/download.rs:229
 msgid "Missing ABS packages "
 msgstr "Недостающие пакеты ABS "
 
-#: src/download.rs:250 src/download.rs:276 src/download.rs:530
-#: src/download.rs:543 src/download.rs:557 src/exec.rs:72 src/exec.rs:113
-#: src/exec.rs:122 src/exec.rs:148 src/exec.rs:161 src/exec.rs:186
-#: src/exec.rs:200 src/exec.rs:221 src/exec.rs:234 src/install.rs:744
-#: src/install.rs:807 src/install.rs:845 src/keys.rs:74
+#: src/download.rs:248 src/download.rs:274 src/download.rs:533
+#: src/download.rs:546 src/download.rs:560 src/exec.rs:82 src/install.rs:788
+#: src/install.rs:851 src/install.rs:889
 msgid "failed to run:"
 msgstr "не удалось выполнить:"
 
-#: src/download.rs:263
+#: src/download.rs:261
 msgid "does not contain PKGBUILD: skipping"
 msgstr "не содержит PKGBUILD — пропускаем"
 
-#: src/download.rs:294
+#: src/download.rs:293
 msgid "downloading: {pkg}"
 msgstr "скачивается: {pkg}"
 
-#: src/download.rs:317
+#: src/download.rs:315
 msgid "Downloading PKGBUILDs..."
 msgstr "Скачиваются PKGBUILD’ы..."
 
-#: src/download.rs:321
+#: src/download.rs:319
 msgid " PKGBUILDs up to date"
 msgstr " PKGBUILD’ы уже последних версий"
 
-#: src/download.rs:520
+#: src/download.rs:523
 msgid "{} is not installed: can not get repo packages"
 msgstr "{} не установлен: не удалось получить пакеты из репозитория"
 
-#: src/download.rs:561
+#: src/download.rs:564
 msgid "asp returned {}"
 msgstr "asp вернул {}"
 
-#: src/exec.rs:86
+#: src/exec.rs:180
 msgid "Pacman is currently in use, please wait..."
 msgstr "В данный момент уже работает Pacman, подождите..."
 
-#: src/fmt.rs:6 src/info.rs:146 src/search.rs:176
+#: src/fmt.rs:6 src/info.rs:151 src/search.rs:214
 msgid "None"
 msgstr "Никаких"
 
-#: src/info.rs:30 src/query.rs:51
+#: src/help.rs:4
+msgid "Usage:"
+msgstr ""
```

Comment:
None of these have been translated

Reply:
indeed. I am too lazy to do it now🙂.

can you merge this and I will translate --help with my girlfriend on next weekend.

Reply:
Right right if it's intentional that's okay, thought you may have missed it. I'll assume the rest is all good then.